js/main.js

```diff
@@ -1,58 +1,106 @@
-/* eslint-disable no-use-before-define */
-// Create array of books
-const bookTitle = document.querySelector('#title');
-const bookAuthor = document.querySelector('#author');
-const bookAdd = document.querySelector('#available-books');
-const btn = document.querySelector('#add-button');
-
-if (!localStorage.getItem('bookInfo')) {
-  localStorage.setItem('bookInfo', JSON.stringify([]));
+/* eslint-disable max-classes-per-file */
+// ======== book class: book representation =========
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
 }
 
-let books;
+// ======= class storage: handle local storage ==========
 
-function saveBooks(book) {
-  localStorage.setItem('bookInfo', JSON.stringify(book));
-}
+class Store {
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    return books;
+  }
+
+  static bookAdd(book) {
+    const books = Store.getBooks();
+    books.push(book);
```

Comment:
- [ ] OPTIONAL
It will be a good practice to use of array methods which doesn't mutate the array, such as `concat` over `push`.